### PR TITLE
Adding VHF tab

### PR DESCRIPTION
**Pull request by Gulfa**

### CODE REVIEW

File: meerkat_frontend/src/files/pages/vhf.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+		</a>
+	</div>
+	<div class="less-padding-col tab-title hidden-xs col-sm-5 col-md-6" id="title">
+	</div>
+	<div id="epi-week-title" class="col-xs-12 col-sm-12 col-md-3 pull-right box ">
+	</div>
+
+</div>
+
+<div class="demographics toggled" id="wrapper">
```
Comment (jonathansberry) [inline meerkat_frontend/src/files/pages/vhf.html]: technically the class "demographics" here should be "vhf" or something uniquely identifying the tab.  Though it is only practically necessary if you want to format something different for this tab only.

File: meerkat_frontend/src/js/technical/tableManager.js [hunk 2]
```diff
@@ -1209,150 +1209,292 @@ function drawAllClinicsCompleteness(containerID, regionID, locations, data) {
...
-                datum.gender = i18n.gettext("Male");
-            }
+    var columns = [{
+      field: "alert_id",
+      title: i18n.gettext('Alert ID'),
+      'searchable': true,
+      width: "10%",
+      valign: "middle"
+    },
+		   {
```
Comment (jonathansberry) [inline meerkat_frontend/src/js/technical/tableManager.js]: Indentation is quite strange here.  Trying to move all this code over to 4 hard spaces like python.  Not that we are going to acheive it any time soon.

File: meerkat_frontend/src/js/technical/tableManager.js [hunk 3]
```diff
@@ -1209,150 +1209,292 @@ function drawAllClinicsCompleteness(containerID, regionID, locations, data) {
...
-            columns: columns,
-            width: "100%",
-            data: data,
-            align: "center",
-            classes: "table table-hover",
-            pagination: true,
-            pageSize: 50
-        });
-        addPaginationListener('#' + containerID + ' table');
+function drawVHFTable(containerID, cases, variables) {
```
Comment (jonathansberry) [inline meerkat_frontend/src/js/technical/tableManager.js]: Any chnce of a short comment explaing what a VHF table is? I presume we are not talking about VHF radio?

Comment (Gulfa) [reply meerkat_frontend/src/js/technical/tableManager.js]: Viral Haemorrhagic Fever

**Pull request merged**